src/main/kotlin/no/nav/familie/oppdrag/domene/OppdragId.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.familie.oppdrag.domene
+
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
+import no.trygdeetaten.skjema.oppdrag.Oppdrag
+
+
+data class OppdragId(val fagsystem : String,
+                     val fødselsnummer : String,
```

Comment:
Kan vi kalle det for `personIdent`? 😄 Så dekker det også Dnr

Reply:
Jepp!